### PR TITLE
Rename the bug report template

### PR DESCRIPTION
**Pull request by MariaNattestad**
(This is from the DeepVariant team, note we are still not taking pull requests at this time.)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-name: Bug report
-about: Create a report to help us improve
+name: 'Problem encountered while running DeepVariant'
```
Comment (gunjanbaid) [inline .github/ISSUE_TEMPLATE/bug_report.md:2]: Not sure if these quotes will show up in the final title. If they do, we can remove as a followup.

**Comment by gunjanbaid** [comment]
LGTM!

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (MariaNattestad) [reply .github/ISSUE_TEMPLATE/bug_report.md:2]: Good question, let's see!

Comment (gunjanbaid) [reply .github/ISSUE_TEMPLATE/bug_report.md:2]: Looks good, no quotes! :)

**Pull request merged**